### ISSUE TITLE
Hab grad meinen ersten Key geclaimed, geht aber nicht.

### ISSUE THREAD

**Issue by LordSN4KE** [Deazl/pr0keys#79]
Moinsen, wollte gerade den Key von "boin" claimen von dem Spiel: "ARK: Survival Evolved"!

Anscheinend hat die kleine Mistratte jedoch einen ungültigen Key reingeschmissen und jetzt kann ich keinen neuen mehr Claimen.

Kann man das irgendwie fixen bitte?

**Comment by Kotter18** [comment]
Habe auch einen ungültigen Rocket League Key von dem Typen bekommen. Der sollte gebannt werden.

**Comment by LordSN4KE** [comment]
Joa, solche Leute sollten sofort gebannt werden.

**Comment by Deazl** [comment]
Der User wurde soeben gebannt. Ich behalte schon im Auge was dort passiert. In den letzten Tagen wurden über 50 User gebannt. Mit ihm zusammen habe ich gerade auch 11 weitere User gebannt. Leider kann ich nicht permanent alles im Auge behalten. Im Update wird jedoch ein AutoBan System implementiert. Anschließend bekomme ich eine Nachricht um das zu prüfen.

**Comment by LordSN4KE** [comment]
Saubere Sache... Gibts eine Möglichkeit, dass ich meine verbrauchte Reputation wieder zurückbekomme, da ich ja schlussendlich keinen Key bekommen habe!

**Comment by nimrahwt** [comment]
yep Ich hab jetzt zum 2ten mal einen ungültigen key bekommen, dieser wahr wohl absichtlich falsch (beim ersten falschen key könnte es sich um ein versehen gehandelt haben laut sharer) , und wieder einen tag unnötiges warten
![capture](https://cloud.githubusercontent.com/assets/18002633/13946105/b1fd4cc4-f012-11e5-8136-c9df83e85e39.PNG)

**Comment by Deazl** [comment]
Ich sitze seit Tagen Stunden am Update um so viele Bugs wie Möglich zu vermeiden. Der größte Teil der Zeit geht beim Testen drauf. Ich hoffe, dass ich spätestens übermorgen die neue Version deployen kann.
Das gröbste ist durch. Einige Feinheiten gibts jedoch noch, welche angepasst werden müssen.

Die größte Herausforderung wird die Datenmigrierung.

In Zukunft wird es möglich sein Keys zu bewerten. Erst nach einer Bewertung (Funktioniert, funktioniert nicht) bekommt auch der jenige der den Key eingereicht hat seine Credits. Wird ein Key 2 Tage lang nicht bewertet, wird er automatisch als "Funktioniert" markiert. Ebenfalls bekommt man seine Credits wieder, wenn ein Key als "Funktioniert nicht" eingereicht wurde. Alles wird jedoch überwacht. Sobald irgendwas auffällt (z.B. wenn ein User mehrfach Keys als "Funktioniert nicht" markiert hat) bekomme ich automatisch eine Benachrichtigung und alle notwendigen Informationen.

**Comment by LordSN4KE** [comment]
Sauber!

**Comment by Moby-Slim** [comment]
Gut, dann warte ich bis zu diesem Update. Ich habe so um die 100+ Humble Bundle Gift links vom letzten Jahr, und weitere 100+ Steam Keys von Bundle Stars/IGN/Gamesrocket, und wo nicht noch alles. Ich ziehe es vor das es die richtigen bekommen, nicht diese(r) Geier.

Auch eine Anfrage an dich, @Deazl. Wärst du so freundlich und löschst Happyator von deazl.com, falls du es nicht schon getan hast. Ist der gescheiterte Versuch gewesen sich, in der 1k Benis Zeit, anzumelden. Habs dann erst mitbekommen das man 1k Benisse braucht, als ich noch mal alles auf pr0 nachgelesen hatte (hatte ich vorher überflogen). Hab auch keine Ahnung mehr was ich als Passwort hatte. Ich fühl mich als Moby-Slim jetzt auf deazl.com sehr wohl. Danke fein im Voraus.

**Comment by NbdY83** [comment]
Leider muss ich mich auch zu Wort melden, heute hat es mich erwischt. Scheint ja ne beliebte Masche zu sein. Wird die "rep" wieder gut geschrieben?

http://i.imgur.com/ttjgN6y.gif

**Comment by evilized84** [comment]
Mich hat es auch gerade erwischt:
Day of the Tentacle Remastered	henci	2016-03-25 01:30:29

Wie geht es nun weiter?

**Comment by NbdY83** [comment]
Anscheinend erstmal gar nicht. Werde keine Keys mehr zur Verfügung stellen solange da nix passiert und meine rep nicht wieder an land kommt. Das ist der Nachteil an diesem System, fand es total schwachsinnig als es eingeführt wurde. Da kann man doch auch direkt in den Steam Trading Foren Keys tauschen.

**Comment by Deazl** [comment]
Wie bereits hier geschrieben (https://github.com/Deazl/pr0keys/issues/79#issuecomment-199961531) können Credits vollautomatisch wieder gutgeschrieben werden.

Hier gibts eine kleine Sneak-Preview dazu:
http://i.imgur.com/2ckKWay.gifv

**Comment by mikeusual** [comment]
Wie is das dann mit den Keys bzw. der Rep. die vor dem Update schon durch falsche Keys draufgegangen ist?

**Comment by Deazl** [comment]
Einfach bewerten.

**Comment by mikeusual** [comment]
Und wenn du die Keys aus der "Claimed Keys" Sektion schon rausgenommen hast? ;)

**Comment by Deazl** [comment]
Inwiefern rausgenommen?

**Comment by mikeusual** [comment]
ich hab 2 mal fake keys gemeldet....rep weg....du hast den user gebannt und die keys aus meiner claimed key sektion rausgenommen....also is ein nachträgliches bewerten nicht mehr möglich...

**Comment by Deazl** [comment]
Ist auch nicht notwendig. Nach dem Update starten alle User mit 0 Credits. Dann können alle User ihre Keys bewerten. Sind Keys 2 Tage lang nicht bewertet worden, werden demjenigen der den Key eingereicht hat seine Credits gutgeschrieben und der Empfänger hat pech gehabt.

**Comment by nimrahwt** [comment]
also  bei mir sieht das fenster ganz anders aus, von bewerten stand da auch nix 
![capture](https://cloud.githubusercontent.com/assets/18002633/14068477/e74fdd26-f484-11e5-92af-a32da7d32f39.PNG)

**Comment by Deazl** [comment]
@nimrahwt Das kommt auch erst im Update :)
Leider sind beim testen noch einige Probleme und Fehler aufgefallen. Besonders was die Performance angeht, weshalb das Update vermutlich erst heute Mittag / Abend statt wie geplant am Samstag erscheint.

**Comment by NbdY83** [comment]
Mit solchen Aussagen bezüglich Updates solltest Du in Zukunft vorsichtiger sein. Wirkt langsam unseriös wenn gefühlt jeder 2. Beitrag von dir mit tollen Neuerungen geschmückt wird und Du deine selbstgelegte Timeline nicht einhalten kannst.

**Comment by Deazl** [comment]
Leider ging ein Geburtstag eines Familienmitglieds gestern unerwartet lange. Da das ganze für 13 Uhr angelegt wurde, habe ich nicht damit gerechnet, dass das ganze bis 18 Uhr geht. Ich habe mich dann "kurz" hingelegt und bin gerade dabei alles vorzubereiten, anzupassen und dann das Update durchzuführen.

**Comment by Deazl** [comment]
So. Mittlerweile hab ich alles auf test.deazl.com einigermaßen zum laufen bekommen. Jetzt fehlen nur noch einige kleine Bugfixes und dann kann es auch schon live gehen.
